nats-server/configuration/websockets.md

```diff
@@ -0,0 +1,60 @@
+# Websocket Support
+
+*Supported since NATS server version 2.2*
+
+Websocket support can be enabled in the server and may be used alongside the
+traditional TCP socket connections.  TLS, compression and
+Origin Header checking are supported.
+
+To enable websocket support in the server, add a `websockets` configuration
+block in the server's configuration file like the following:
+
+```
+websocket {
+    # Specify a host and port to listen for websocket connections
+    # listen: "host:port"
+
+    # It can also be configured with individual parameters,
+    # namely host and port.
+    # host: "hostname"
+    # port: 4443
+
+    # This will optionally specify what host:port for websocket
+    # connections to be advertised in the cluster
+    # advertise: "host:port"
+
+    # TLS configuration is required
+    tls {
+      cert_file: "/path/to/cert.pem"
+      key_file: "/path/to/key.pem"
+    }
+
+    # If same_origin is true, then the Origin header of the
+    # client request must match the request's Host.
+    # same_origin: true
+
+    # This list specifies the only accepted values for
+    # the client's request Origin header. The scheme,
+    # host and port must match. By convention, the
+    # absence of port for an http:// scheme will be 80,
```

Comment:
What does "...the absence of port..." mean in this context?

Reply:
I am going to update this PR with some missing params and fix that.